scripts/toolchain.sh

```diff
@@ -45,7 +45,7 @@ DEFAULT_CLANG_TIDY_VERSION=13
 export CLANG_TIDY="${CLANG_TIDY:-$(which "clang-tidy-$DEFAULT_CLANG_TIDY_VERSION" || which clang-tidy)}"
 
 if [ ! "$CLANG_TIDY" ]; then
-    printf '\033[33m[WARNING] Clang Tidy ('"$DEFAULT_CLANG_TIDY_VERSION"') has not been installed.\033[0m\n' >&2
+    printf '\033[33m[WARNING] Clang Tidy (%d) has not been installed.\033[0m\n' "$DEFAULT_CLANG_TIDY_VERSION" >&2
```

Comment:
Naming recommendation: CLANG_TIDY_DEFAULT_VERSION, so that related var can be grouped together.